src/interface/src/app/plan/create-scenarios/create-scenarios.component.ts

```diff
@@ -0,0 +1,132 @@
+      ),
+      state(
+        'collapsed',
+        style({
+          backgroundColor: '#ebebeb',
+          padding: '0px',
+          width: '36px',
+        })
+      ),
+      transition('expanded => collapsed', [
+        group([
+          query('@expandCollapseButton', animateChild()),
+          query('@expandCollapsePanelContent', animateChild()),
+          animate('300ms 100ms ease-out'),
+        ]),
+      ]),
+      transition('collapsed => expanded', [
+        group([
+          query('@expandCollapseButton', animateChild()),
+          query('@expandCollapsePanelContent', animateChild()),
+          animate('250ms ease-out'),
+        ]),
+      ]),
+    ]),
+    trigger('expandCollapseButton', [
+      state(
+        'expanded',
+        style({
+          backgroundColor: 'white',
+        })
+      ),
+      state(
+        'collapsed',
+        style({
+          backgroundColor: '#ebebeb',
+        })
+      ),
+      transition('expanded => collapsed', [animate('300ms ease-out')]),
+      transition('collapsed => expanded', [animate('250ms ease-out')]),
+    ]),
+    trigger('expandCollapsePanelContent', [
+      state(
+        'expanded',
+        style({
+          opacity: 1,
+        })
+      ),
+      state(
+        'collapsed',
+        style({
+          opacity: 0,
+        })
+      ),
+      transition('expanded => collapsed', [animate('100ms ease-out')]),
+      transition('collapsed => expanded', [animate('100ms 250ms ease-out')]),
+    ]),
+  ],
+})
+export class CreateScenariosComponent implements OnInit {
+  @Input() plan$ = new BehaviorSubject<Plan | null>(null);
+
+  readonly text1: string = `
+    Scenarios consist of project areas identified by your priorities and constraints
+    within the planning area. You can draw or upload your own project areas when creating
+    scenarios or Planscape can recommend some project areas and scenarios within those
+    project areas.
+  `;
+
+  readonly text2: string = `
+    You can choose to use either Current Condition scores or Management Opportunity scores to
+    inform your selection of priorities.
+  `;
+
+  readonly text3: string = `
+    Define project areas based on choosing priorities based only on the current condition of
+    the defined planning area. Future modeling is not considered in this mode.
+  `;
+
+  readonly text4: string = `
+    Choose priorities using the Pillars of Resilience Framework. Priorities with higher
+    opportunity scores (-1 to +1 range) represent how management value is greatest in the near
+    term within the defined planning area. Future modeling is considered in this mode.
+  `;
+
+  readonly text5: string = `
+    To learn more about how priorities are defined and used within this tool, visit
+    linkaddresshere.
+  `;
+
+  legend: Legend | undefined;
+  panelExpanded: boolean = true;
+
+  constructor(private mapService: MapService) {}
+
+  ngOnInit(): void {
+    this.mapService
+      .getColormap('viridis')
```

Comment:
optional: We could make 'viridis' and other colormap names into an enum

Reply:
This is a good idea, but IMO we can wait to implement it until we know the final colormap(s) that we're using. 'viridis' is just a placeholder here for now and I'll add a TODO to make that clear.